class AuthenticationError(Exception):
    """Exceção para falhas na autenticação."""
    pass